Core/src/Material/Interactions.cpp

```diff
@@ -175,56 +214,51 @@ float Acts::deriveEnergyLossBetheQOverP(const MaterialSlab& slab,
   //     + eps * (d(u)/(u) + d(wmax)/(wmax) - 2 * d(beta²) - d(delta))
   //
   // where we can use d(eps) = eps * (d(eps)/eps) for further simplification.
-  const auto logDerEps = logDeriveEpsilon(qOverP, rq);
-  const auto derDHalf = deriveDeltaHalf(qOverP, rq);
-  const auto logDerU = logDeriveMassTerm(qOverP);
-  const auto logDerWmax = logDeriveWMax(m, qOverP, rq);
-  const auto derBeta2 = deriveBeta2(qOverP, rq);
-  const auto rel = logDerEps * (std::log(u / I) + std::log(wmax / I) -
-                                2.0f * rq.beta2 - 2.0f * dhalf) +
-                   logDerU + logDerWmax - 2.0f * derBeta2 - 2.0f * derDHalf;
+  const float logDerEps = logDeriveEpsilon(qOverP, rq);
+  const float derDHalf = deriveDeltaHalf(qOverP, rq);
+  const float logDerU = logDeriveMassTerm(qOverP);
+  const float logDerWmax = logDeriveWMax(m, qOverP, rq);
+  const float derBeta2 = deriveBeta2(qOverP, rq);
+  const float rel = logDerEps * (std::log(u / I) + std::log(wmax / I) -
+                                 2.0f * rq.beta2 - 2.0f * dhalf) +
+                    logDerU + logDerWmax - 2.0f * derBeta2 - 2.0f * derDHalf;
   return eps * rel;
 }
 
-float Acts::computeEnergyLossLandau(const MaterialSlab& slab, int /* unused */,
-                                    float m, float qOverP, float q) {
-  ASSERT_INPUTS(m, qOverP, q)
-
+float Acts::computeEnergyLossLandau(const MaterialSlab& slab, float m,
+                                    float qOverP, float absQ) {
   // return early in case of vacuum or zero thickness
   if (not slab) {
     return 0.0f;
   }
 
-  const auto I = slab.material().meanExcitationEnergy();
-  const auto Ne = slab.material().molarElectronDensity();
-  const auto thickness = slab.thickness();
-  const auto rq = Acts::RelativisticQuantities(m, qOverP, q);
-  const auto eps = computeEpsilon(Ne, thickness, rq);
-  const auto dhalf = computeDeltaHalf(I, Ne, rq);
-  const auto u = computeMassTerm(Me, rq);
+  const auto rq = RelativisticQuantities(m, qOverP, absQ);
```

Comment:
It's a style choice, but I personally don't like `auto` when the standard construction syntax could be used.

Arguably this could be

```cpp
const RelativisticQuantities rq{m, qOverP, absQ};
```

Also in some cases below.

Reply:
right, I left this as is but I can adopt that